js/uid2ApiClient.ts

```diff
@@ -0,0 +1,138 @@
+interface IdentityBase {
```

Comment:
License?

Reply:
I'll check all the files have a license at the end - hopefully my suggestion about possibly reducing/skipping license information in individual files might go somewhere before I get this merged back to the main repo.

I'll leave this open until then to make sure I don't forget.